components/ui/service-docs-page.tsx

```
@@ -0,0 +1,41 @@
+export function ServiceDocsPage({ page, children, ...props }: ServiceDocsPageProps) {
+    // Instead of context, we'll modify the children to inject our icon
+    let modifiedChildren = children;
+
+    // If service is defined, we'll create our icon
+    if (page.data.service) {
+        const serviceIcon = <ServicePageIcon page={page} className="inline-block mr-2 h-6 w-6" />;
+
+        // If the first child is the title, we'll add our icon to it
+        if (Array.isArray(children) && children.length > 0) {
+            const firstChild = children[0];
+            if (firstChild && firstChild.type && firstChild.type.name === 'DocsTitle') {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)